src/common.js

```diff
@@ -4,11 +4,23 @@
 
 var os = require('os');
 var fs = require('fs');
+var path = require('path');
 var glob = require('glob');
 var shell = require('..');
 
 var DEFAULT_ERROR_CODE = 1;
 var shellMethods = Object.create(shell);
+var PATH = 'PATH';
+
+// windows calls it's path 'Path' usually, but this is not guaranteed.
```

Comment:
@gyandeeps if this is not the case, then it would be good to chime in on nodejs/node#9157 and the corresponding pull request before Node 7 gets incorrect documentation.

As for this PR, let's go with your initial approach. Sorry for the confusion.